webapp/locales/en.json

```diff
@@ -329,6 +329,28 @@
       "mute": "Mute user",
       "unmute": "Unmute user",
       "unmuted": "{name} is unmuted again"
+    },
+    "blocked-users": {
+      "name": "Blocked users",
+      "explanation": {
+        "intro": "If another user has been blocked by you, this is what happens:",
+        "your-perspective": "You will no longer be able to interact with their contributions.",
+        "their-perspective": "Vice versa: The blocked person will also no longer be able to interact with your contributions.",
+        "notifications": "Blocked users will no longer receive notifications if they mention each other.",
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-01-23T17:22:46Z" title="Thursday, January 23rd 2020, 6:22:46 pm +01:00">Jan 23, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -329,6 +329,28 @@
       "mute": "Mute user",
       "unmute": "Unmute user",
       "unmuted": "{name} is unmuted again"
+    },
+    "blocked-users": {
+      "name": "Blocked users",
+      "explanation": {
+        "intro": "If another user has been blocked by you, this is what happens:",
+        "your-perspective": "You will no longer be able to interact with their contributions.",
+        "their-perspective": "Vice versa: The blocked person will also no longer be able to interact with your contributions.",
+        "notifications": "Blocked users will no longer receive notifications if they mention each other.",
```


Is that actually true? I suggested to remove it from the German translations, see above :point_up:

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-23T17:55:15Z" title="Thursday, January 23rd 2020, 6:55:15 pm +01:00">Jan 23, 2020</time>_

---


yes, see my response :point_up_2: